Taggers/test/MVATraining/checkJsons.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import os
+
+sys.path.append('/cvmfs/cms.cern.ch/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_26/bin/slc6_amd64_gcc530/')
```

Comment:
???

Reply:
The whole MVATraining directory is ancient. I think it can go completely

Reply:
I agree

Reply:
Removed it from my branch